pkg/ebpf/event_common.go

```diff
@@ -108,8 +112,17 @@ func (c ConnectionStats) String() string {
 		c.Direction,
 		c.MonotonicSentBytes, c.LastSentBytes,
 		c.MonotonicRecvBytes, c.LastRecvBytes,
-		c.MonotonicRetransmits, c.LastRetransmits,
 	)
+
+	if c.Type == TCP {
+		str += fmt.Sprintf(
+			", %d retransmits (+%d), RTT %s",
+			c.MonotonicRetransmits, c.LastRetransmits,
+			time.Duration(c.RTT)*time.Microsecond,
```

Comment:
Should we include variance as well?

Reply:
`RTT %s (± %s)`

Reply:
TBH I'm not sure if this is a super rigorous way to represent it, but I think it's a nice to have. I noticed that `ss -i -t` tool prints out `rtt/rtt_var` but I find it even more confusing.